docs/manuals/measurement/readout_mitigation.rst

```diff
@@ -143,16 +130,31 @@ Expectation value
 
 .. jupyter-execute::
 
+    def str2diag(string):
```

Comment:
Why we have to define this function although you port this function to `.data_processing.mitigation.utils`?

Instead of using the util function, we can also write:
```python
from qiskit.quantum_info import Operator
Operator.from_label("II0I").to_matrix()
```
which looks much simpler.

Reply:
I didn't want to make `.data_processing.mitigation.utils` public and `str2diag` was the only code that I did not know how to replace with other public code. I thought there was probably something in Qiskit but I had not figured it out. I can try `Operator.from_label`.

Reply:
@nkanazawa1989 `Operator.from_label` worked. This is ready for review now.